src/client/src/commons/search/editor/searchEditorState.js

```diff
@@ -88,6 +88,9 @@ const initialState = {
     // Chronostratigraphy filter
     chronostratigraphy_id: null,
 
+    // Chronostratigraphy filter
```

Comment:
REQ: Kommentar anpassen
```suggestion
    // Lithostratigraphy filter
```